.github/workflows/ci.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
       fail-fast: true
       matrix:
         os: [ubuntu-latest]
-        ghc: ['8.0', '8.2', '8.4', '8.6', '8.8', '8.10', '9.0', '9.2']
+        ghc: ['8.0', '8.2', '8.4', '8.6', '8.8', '8.10', '9.0', '9.2', '9.4.1']
```

Comment:
Since you are departing from the _x.y_ scheme for 9.4.1, please add a comment there why you do so.

Reply:
Done.